tests/system/providers/amazon/aws/example_sagemaker.py

```diff
@@ -178,7 +177,8 @@ def _build_and_upload_docker_image(preprocess_script, repository_uri):
         if docker_build.returncode != 0:
             raise RuntimeError(
                 "Failed to prepare docker image for the preprocessing job.\n"
-                f"The following error happened while executing the sequence of bash commands:\n{stderr}"
+                "The following error happened while executing the sequence of bash commands:\n"
+                f"{stderr.decode()}"
```

Comment:
stderr is a byte array. Without the `decode()`, the `\n` are not escaped for instance, and the whole output appears on one line, which is _not the best_